quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/cli/Help.java

```diff
@@ -58,16 +55,12 @@ public String optionList(Layout layout, Comparator<OptionSpec> optionSort, IPara
     private List<OptionSpec> excludeHiddenAndGroupOptions(List<OptionSpec> all) {
         List<OptionSpec> result = new ArrayList<>(all);
 
-        for (ArgGroupSpec group : optionSectionGroups()) { result.removeAll(group.allOptionsNested()); }
-
-        for (Iterator<OptionSpec> iter = result.iterator(); iter.hasNext(); ) {
-            OptionSpec optionSpec = iter.next();
-
-            if (!isVisible(optionSpec)) {
-                iter.remove();
-            }
+        for (ArgGroupSpec group : optionSectionGroups()) {
+            result.removeAll(group.allOptionsNested());
         }
 
+        result.removeIf(optionSpec -> !isVisible(optionSpec));
```

Comment:
To keep the code consistent, we should either leave it as is or use a method reference here.